com.minres.coredsl/src/com/minres/coredsl/scoping/CoreDslScopeProvider.xtend

```diff
@@ -56,25 +56,15 @@ class CoreDslScopeProvider extends AbstractCoreDslScopeProvider {
             }
         } else if(reference.EReferenceType == CoreDslPackage.Literals.DIRECT_DECLARATOR) {
             val parent = context.eContainer
-            if(parent instanceof PostfixExpression) {
+            // TODO for some reason, parent.directDeclarator.eContainer is null here
```

Comment:
The reason is that you resolve it inside-out. Therefore the parrent is not yet resolved...